### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by 12721067**
目前是依照之前給的collection寫

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+  -X POST \
+  -H 'content-type: application/json' \
+  -d '{
+	    "username": "Vicky",
+        "email": "lch82327@gmail.com",
+        "password": "v123456",
+        "first_name": "Chi-hsuan" ,
+        "last_name": "Li" 
+   }'
+```
```
Comment (vicky-sunshine) [inline README.md]: Please use fake information here

File: README.md [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+
+### Get Account Information
+
+#### Example
+
+**GET /api/v2/accounts/{user_id}**
+```shell
+$ curl http://localhost:9292/api/v2/accounts/{user_id} \
+    -X GET \
```
Comment (vicky-sunshine) [inline README.md]: It should be `{user_username}` here

File: test.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+>1
```
Comment (vicky-sunshine) [inline test.md]: This file is redundant, please remove this file

File: README.md [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+
+Run this API by using: 
+
+```
+$ rackup
+```
+
+### Set Keys
+
+For the first-time execution, it's necessary to new a "config_env.rb" file in "config" folder.(refer to config_env.rb.example)
```
Comment (vicky-sunshine) [inline README.md]: use ' `` ' replace ' "" '
ex: `config_env.rb`

File: README.md [hunk 5]
```diff
@@ -0,0 +1,357 @@
...
+}
+```
+
+### Get Containers Information of an Account
+
+#### Example
+
+**GET /api/v2/accounts/{user_id}/containers**
+```shell
+$ curl http://localhost:9292/api/v2/accounts/{user_id}/containers \
```
Comment (vicky-sunshine) [inline README.md]: It should be `{user_username}` here

File: README.md [hunk 6]
```diff
@@ -0,0 +1,357 @@
...
+```
+[]
+```
+### Create a Container
+
+#### Example
+
+**POST /api/v2/accounts/{user_id}/containers**
+```shell
+$ curl http://localhost:9292/api/v2/accounts/{user_id}/containers \
```
Comment (vicky-sunshine) [inline README.md]: It should be `{user_username}` here

File: README.md [hunk 7]
```diff
@@ -0,0 +1,357 @@
...
+    "public_ip": null
+  }
+}
+```
+
+### Get a Certain Container of an Account
+
+#### Example
+
+**GET /api/v2/accounts/{user_id}/containers/{container_id}**
```
Comment (vicky-sunshine) [inline README.md]: It should be `{user_username}` here

File: README.md [hunk 8]
```diff
@@ -0,0 +1,357 @@
...
+    "public_ip": null
+  }
+}
+```
+
+### Update Route Info of Certain Container for an Account
+
+#### Example
+
+**POST /api/v2/accounts/{user_id}/containers/{container_id}/route**
```
Comment (vicky-sunshine) [inline README.md]: It should be `{user_username}` here

File: README.md [hunk 9]
```diff
@@ -0,0 +1,357 @@
...
+  "default_gateway": null,
+  "local_network": null
+}
+```
+
+### Run Up NAT Service on Certain Container of an Account
+
+#### Example
+
+**POST /api/v2/accounts/{user_id}/containers/{container_id}/services**
```
Comment (vicky-sunshine) [inline README.md]: It should be `{user_username}` here

File: README.md [hunk 10]
```diff
@@ -0,0 +1,357 @@
...
+    "public_ip": null
+  }
+}
+```
+
+### Stop NAT Service on Certain Container of an Account
+
+#### Example
+
+**POST /api/v2/accounts/{user_id}/containers/{container_id}/services**
```
Comment (vicky-sunshine) [inline README.md]: It should be `{user_username}` here

File: README.md [hunk 11]
```diff
@@ -0,0 +1,357 @@
...
+    "public_ip": null
+  }
+}
+```
+
+### Delete Certain Containers Infos For an Account
+
+#### Example
+
+**DELETE /api/v2/accounts/{user_id}/containers/{container_id}**
```
Comment (vicky-sunshine) [inline README.md]: It should be `{user_username}` here

**Comment by 12721067** [comment]
update: 
changed `username` to `John Doe`
removed `test.md`

**Pull request merged**